src/main/java/org/gradle/profiler/mutations/ApplyProjectDependencyChangeMutatorConfigurator.java

```diff
@@ -0,0 +1,71 @@
+package org.gradle.profiler.mutations;
+
+import com.google.common.collect.ImmutableSet;
+import com.typesafe.config.Config;
+import org.gradle.profiler.BuildMutator;
+import org.gradle.profiler.CompositeBuildMutator;
+import org.gradle.profiler.InvocationSettings;
+import org.gradle.profiler.mutations.support.ProjectCombinations;
+
+import java.io.File;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static org.gradle.profiler.mutations.support.ProjectCombinationsSupport.createProjectCombinations;
+import static org.gradle.profiler.mutations.support.ScenarioSupport.sourceFiles;
+
+public class ApplyProjectDependencyChangeMutatorConfigurator implements BuildMutatorConfigurator {
```

Comment:
This should be a nested `Configurator` class inside `ApplyProjectDependencyChangeMutator`.

Reply:
What is the reason for that? There is for example FileChangeMutatorConfigurator that is not nested. And nesting this class will make ApplyProjectDependencyChangeMutator bigger and harder to read.

Reply:
Ok, I moved it to the `ApplyProjectDependencyChangeMutator`